compiler/compile.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Other groups (see pubCrawl) had this script to make compilation easier for the user. We can remove it, but I was really confused on how to implement and run odds.ml. This script simplifies it for the user.